### ISSUE TITLE
Multiple selection and use it in fragments

### ISSUE THREAD

**Issue by fesave** [TheLuckyCoder/MaterialChooser#5]
Hello, have you thought about the possibility of adding multiple selection for the folders? and The possibility of being able to use the `onActivityResult()` in a fragment without depending on the activity? Thank you.

**Comment by razvanfilea** [comment]
Yes! I recently started exploring the new [Result API](https://developer.android.com/training/basics/intents/result) but it's still in alpha for now. This new API also works in Fragments as well as Activities. You could give it a try and see if you can make it work from your end for now.

**Comment by fesave** [comment]
Okay! I will follow your news, thanks for responding so quickly. Cheers!
